File: intcbbta-rest/src/main/java/com/isc/intern/cbbta/bean/UserInfoResponse.java
package com.isc.intern.cbbta.bean;

import java.util.Date;

public class UserInfoResponse extends BaseResponse{

	private static final long serialVersionUID = 1L;
	private Long recId;
    private String employeeNo;
    private String fname;
    private String lname;
    private String image;
    private String mobileNo;
    private String role;
    private String email;
    private String faceRegisStatus;
    private String faceStatus;
	
	public Long getRecId() {
		return recId;
	}
	public void setRecId(Long recId) {
		this.recId = recId;
	}
	public String getEmployeeNo() {
		return employeeNo;
	}
	public void setEmployeeNo(String employeeNo) {
		this.employeeNo = employeeNo;
	}
	public String getFname() {
		return fname;
	}
	public void setFname(String fname) {
		this.fname = fname;
	}
	public String getLname() {
		return lname;
	}
	public void setLname(String lname) {
		this.lname = lname;
	}
	public String getImage() {
		return image;
	}
	public void setImage(String imagePath) {
		this.image = imagePath;
	}
	public String getMobileNo() {
		return mobileNo;
	}
	public void setMobileNo(String mobileNo) {
		this.mobileNo = mobileNo;
	}
	public String getRole() {
		return role;
	}
	public void setRole(String role) {
		this.role = role;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}
	public String getFaceRegisStatus() {
		return faceRegisStatus;
	}
	public void setFaceRegisStatus(String faceRegisStatus) {
		this.faceRegisStatus = faceRegisStatus;
	}
	public String getFaceStatus() {
		return faceStatus;
	}
	public void setFaceStatus(String faceStatus) {
		this.faceStatus = faceStatus;
	}
	public static long getSerialversionuid() {
		return serialVersionUID;
	}
}

File: DemoFlutter/rest_package/lib/bean_user/user_info_response.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:rest_package/bean_user/base_response.dart';
part 'user_info_response.g.dart';

@JsonSerializable()
class UserInfoResponse extends BaseResponse {
  var recId;
  String employeeNo;
  String fname;
  String lname;
  String image;
  String mobileNo;
  String role;
  String email;
  String faceRegisStatus;
  String faceStatus;
  UserInfoResponse(
      this.recId,
      this.employeeNo,
      this.fname,
      this.lname,
      this.image,
      this.mobileNo,
      this.role,
      this.email,
      this.faceRegisStatus,
      this.faceStatus,
      String respCode,
      String respDesc,
      String reqRefNo,
      String respRefNo)
      : super(respCode, respDesc, reqRefNo, respRefNo);
  factory UserInfoResponse.fromJson(Map<String, dynamic> json) =>
      _$UserInfoResponseFromJson(json);
  Map<String, dynamic> toJson() => _$UserInfoResponseToJson(this);
}
